pallets/committee/src/lib.rs

```diff
@@ -722,6 +722,11 @@ impl<
             r => Err(O::from(r)),
         })
     }
+
+    #[cfg(feature = "runtime-benchmarks")]
+    fn successful_origin() -> O {
```

Comment:
Can we please avoid using `O` (Oh)? It is easy to confuse with `0`(Zero).

Reply:
We can avoid, But it is inherited from the substrate that's why I avoid changing them